docs/gettinginvolved/acceptance-testing.rst

```diff
@@ -223,6 +235,70 @@ The nodes should be configured to allow key based SSH connections as user ``root
 
    admin/run-acceptance-tests --distribution centos-7 --provider managed --config-file config.yml
 
+If you are using the ``managed`` provider with ``AWS`` nodes, you  will need to supply both the private and public IP addresses for each node.
+AWS nodes do not have public IP addresses configured in the operating system; instead Amazon routes public IP traffic using NAT.
+In this case the acceptance tests need a hint in order to map the private IP address reported by the Flocker ``/state/nodes`` API to the public node address.
```

Comment:
Maybe (next, not instead of this) we should teach Flocker on AWS to discover and report public addresses on its own.  We already kinda know most clients of Flocker are going to have to do this public/private mapping thing - and this is even the second time we're doing it ourselves.  Are we going to have to repeat this for, say, the benchmarking tool?